saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java

```diff
@@ -67,4 +78,46 @@ public void deleteJudgmentsWithoutRawSourceJudgment(Class<? extends Judgment> ju
         }
     }
     
+    @Transactional
+    public void deleteMeansOfAppealWithoutJudgments(CourtType courtType) {
```

Comment:
I wonder if we shouldn't move this service to the import module. The same as it has been done with ScObjectDeleter.